api/models/Labels.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+  attributes: {
+    name: { type: 'string', required: true, unique: true },
+    hotKey: { type: 'string' },
```

Comment:
should this also have `unique: true`?

Reply:
It's not unique because when a label has no hotkey, this hotKey's value is "". But you are right, I'm considering to change the logic, if it has no hotkey, just leave it to be no value, so that the "unique" constraint will not be violated. That's a better way to do this.